deployment/modules/gcp/storage/main.tf

```diff
@@ -65,3 +65,47 @@ resource "google_spanner_database" "dedup_db" {
     "CREATE TABLE IDSeq (id INT64 NOT NULL, h BYTES(MAX) NOT NULL, idx INT64 NOT NULL,) PRIMARY KEY (id, h)",
   ]
 }
+
+# Secret Manager
+
+# ECDSA key with P256 elliptic curve
+resource "tls_private_key" "sctfe-ecdsa-p256" {
```

Comment:
AIUI this will leave this private key stored in the TF state unencrypted.

With our CI setups, this is less dangerous since the state is remote and encrypted at rest, but definitely worth a comment to this effect in case someone comes along and cut'n'pastes bits of this for their own TF config.

Reply:
Added the security notice.